test/api/v3/integration/user/POST-user_class_cast_spellId.test.js

```diff
@@ -161,6 +161,40 @@ describe('POST /user/class/cast/:spellId', () => {
       });
   });
 
+  it('Issue #12361: returns an error if chilling frost has already been cast', async () => {
+    await user.update({
+      'stats.class': 'wizard',
+      'stats.lvl': 15,
+      'stats.mp': 400,
+      'stats.buffs.streaks': true,
+    });
+    await user.sync();
+    await expect(user.post('/user/class/cast/frost'))
+      .to.eventually.be.rejected.and.eql({
+        code: 401,
```

Comment:
Why is this one a status 401 while the other is 400? 400 seems more appropriate.